src/stores/login.ts

```
@@ -1,8 +1,7 @@
 import { create } from 'zustand'
 import { persist } from 'zustand/middleware'
 import Cookies from 'js-cookie'
-import { isLikelyValidToken } from '@/lib/jwt'
-
+import { useUserStore } from '@/stores/user'
```

ESLint

[error] 'useUserStore' is defined but never used.

(@typescript-eslint/no-unused-vars)